### DISCUSSION TITLE
Android

### DISCUSSION THREAD

**Discussion by DaRealMonika** [L0615T1C5-216AC-9437/MaxRateCalculator#4] [Q&A]
Will you ever add support for android users? as i have no desktop device only an android device and i would love to use this mod.

**Comment by L0615T1C5-216AC-9437** [comment]
I *could* do it but I would have no way to test it and I have never made a mod compatible with android before.

**Reply by DaRealMonika** [reply]
well i can think of 2 solutions to this either try to get like BlueStacks or let me do the testing (with this last method you would have to do the programming as I'm not very good at making minedustry mods) although it'd be easier to do the former as then you could do it all yourself and on 1 device

**Reply by L0615T1C5-216AC-9437** [reply]
sounds like a whole lot of work

**Reply by DaRealMonika** [reply]
well it would be a bit of work that's for sure but if you did option 1 then you could be able to make more mods and have them also support android also just wondering what part of the code makes it not work on android?
